src/reducers/feedReducerDB.js

```diff
@@ -1,6 +1,9 @@
 const initialState = {
   loading: false,
+  switchLoad: false,
```

Comment:
I feel like the feed reducer is getting a bit big but since a lot of the values depend on feed, not sure how to divide this lol

Reply:
You're right, I suppose we could create a separate reducer for things like loading or error states.